client/blocks/importer/wordpress/upgrade-plan/index.tsx

```diff
@@ -137,6 +145,10 @@ export const UpgradePlan: React.FunctionComponent< Props > = ( props: Props ) =>
 				'Migrations are exclusive to the Creator plan. Check out all its benefits, and upgrade to get started.'
 		  );
 
+	if ( isFetchingHostingDetails ) {
```

Comment:
Should we also test if the plans are loading?

In my tests, I didn't see anything weird, but maybe it's because that usually the plans are coming faster than the hosting details?

Reply:
As we talked in Slack, based on [this](https://github.com/Automattic/wp-calypso/blob/54e8be25a5056f8f1b709ff93139de03037ad861/packages/data-stores/src/plans/hooks/use-pricing-meta-for-grid-plans.ts#L107-L113), it seems fine to check it as `! pricing`, as you did in https://github.com/Automattic/wp-calypso/pull/92201/commits/850481c55c5be4335e48bf0b20788fe6cc36a62c.

But maybe @chriskmnds could double-check to confirm it? :)

Reply:
> But maybe @chriskmnds could double-check to confirm it? :)

I think it is not critical and shouldn't block this PR, we can address feedback in a follow-up PR if any.